### PR TITLE
[15.0][MIG] sale_order_product_availability_inline: Migration to 15.0 

### PR DESCRIPTION
**Pull request by cesar-tecnativa**
@Tecnativa TT36524

### CODE REVIEW

**Comment by cesar-tecnativa** [comment]
@sergio-teruel please review

File: sale_order_product_availability_inline/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2020 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Sale Order Product Availability Inline",
+    "summary": "Show product availability in sales order line product " "drop-down.",
+    "version": "15.0.0.0.1",
+    "development_status": "Beta",
```
Comment (sergio-teruel) [inline sale_order_product_availability_inline/__manifest__.py]: Mybe you can change status to maturity??

**Review by sergio-teruel — CHANGES_REQUESTED** [review]
base_view_inheritance_extension is failing??

File: sale_order_product_availability_inline/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2020 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Sale Order Product Availability Inline",
+    "summary": "Show product availability in sales order line product " "drop-down.",
+    "version": "15.0.0.0.1",
```
Comment (pedrobaeza) [inline sale_order_product_availability_inline/__manifest__.py]:
```suggestion
    "version": "15.0.1.0.0",
```

**Comment by rousseldenis** [comment]
@cesar-tecnativa Could you check tests ?

File: sale_order_product_availability_inline/__manifest__.py (hunk 1 above)
Comment (cesar-tecnativa) [reply sale_order_product_availability_inline/__manifest__.py]: Done

File: sale_order_product_availability_inline/__manifest__.py (hunk 2 above)
Comment (cesar-tecnativa) [reply sale_order_product_availability_inline/__manifest__.py]: Done

**Comment by pedrobaeza** [comment]
Please check tests

**Comment by cesar-tecnativa** [comment]
> @cesar-tecnativa Could you check tests ?
waiting for 
https://github.com/OCA/server-tools/pull/2365

**Comment by cesar-tecnativa** [comment]
@chienandalu could you please review this?

File: sale_order_product_availability_inline/models/product_product.py [hunk 3]
```diff
@@ -11,7 +11,7 @@ def name_get(self):
         if self.env.context.get("so_product_stock_inline"):
             res = super().name_get()
             self = self.with_context(warehouse=self.env.context.get("warehouse"))
-            availability = {r.id: [r.free_qty, r.uom_id.name] for r in self}
+            availability = {r.id: [r.free_qty, r.uom_id.name_get()[0][1]] for r in self}
```
Comment (chienandalu) [inline sale_order_product_availability_inline/models/product_product.py]: Why this change? In any case, why not using `display_name`. A tip from the code:

https://github.com/odoo/odoo/blob/c79bda62edd8316c2a5754d8aa8838f8aa4caee0/odoo/models.py#L1821

**Review by chienandalu — CHANGES_REQUESTED** [review]
Thanks, @cersar-tecnativa Tested ok. Just a little remark

**Review by chienandalu — APPROVED** [review]

**Comment by cesar-tecnativa** [comment]
@LudLaf Please review

**Comment by cesar-tecnativa** [comment]
@CarlosRoca13 can you review this?

**Review by CarlosRoca13 — APPROVED** [review]

File: sale_order_product_availability_inline/tests/test_sale_order_product_availability_inline.py [hunk 4]
```diff
@@ -10,8 +10,9 @@ def setUpClass(cls):
         super().setUpClass()
         cls.obj_product = cls.env["product.product"]
         cls.obj_partner = cls.env["res.partner"]
-        cls.obj_inventory = cls.env["stock.inventory"]
+        cls.obj_quant = cls.env["stock.quant"]
         cls.partner = cls.obj_partner.create({"name": "Partner test - sopai"})
+
```
Comment (pedrobaeza) [inline sale_order_product_availability_inline/tests/test_sale_order_product_availability_inline.py]: Don't introduce empty lines in the middle of a method if they don't previously exist

File: sale_order_product_availability_inline/views/sale_views.xml [hunk 5]
```diff
@@ -10,32 +10,22 @@
...
-                    name="context"
-                    operation="python_dict"
-                    key="warehouse"
-                >parent.warehouse_id</attribute>
+                <attribute name="context" operation="update">
+                    {"so_product_stock_inline": True,
+                    "warehouse": parent.warehouse_id}
+                </attribute>
             </xpath>
+
```
Comment (pedrobaeza) [inline sale_order_product_availability_inline/views/sale_views.xml]: Remove these extra empty lines

**Comment by LudLaf** [comment]
> @LudLaf Please review

Modified USAGE.rst file to clarify instructions.

**Review by pedrobaeza — APPROVED** [review]
